ios/engine/KMEI/KeymanEngine.xcodeproj/project.pbxproj

```diff
@@ -275,13 +275,6 @@
 			remoteGlobalIDString = C06D372A1F81F4E100F61AE0;
 			remoteInfo = KeymanEngine;
 		};
-		CE4459CB23FBB8DA003151FD /* PBXContainerItemProxy */ = {
-			isa = PBXContainerItemProxy;
-			containerPortal = F243887514BBD43000A3E055 /* Project object */;
-			proxyType = 1;
-			remoteGlobalIDString = F243887D14BBD43000A3E055;
-			remoteInfo = KeymanEngineDemo;
-		};
```

Comment:
Fixes build dependency / scheme setup for the unit-test target, which relies upon `KeymanEngineTestHost`, not `KeymanEngineDemo`.  The former has an entry that's being left in place, but we need to remove this one.